### PR TITLE
fix #78651 pikabu.ru

### LINKED ISSUE #78651: pikabu.ru
### Issue URL (Ads)
 [https://pikabu.ru/story/ublyudok_na_gazeli_8104030#comments](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fpikabu.ru%2Fstory%2Fublyudok_na_gazeli_8104030%23comments)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.1.0
Browser: | Safari
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion<br/><br/><b>Language-specific:</b><br/> AdGuard Russian, <br/>AdGuard Dutch
Custom filters: | www.pikabu.ru,<br>www.yandex.ru,<br>www.youtube.com

### PR DESCRIPTION
**Pull request by stanislav-atr**
#78651

### CODE REVIEW

File: RussianFilter/sections/specific.txt [hunk 1]
```diff
@@ -59,6 +59,7 @@ agroter.com.ua##.site-main > img
 cripo.com.ua###ai_widget-2
 miasskiy.ru##.papexreka
 miasskiy.ru##.papexrec-adlabel
+pikabu.ru##.stories-feed__message
```
Comment (Alex-302) [inline RussianFilter/sections/specific.txt:62]: image
ломаешь

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: RussianFilter/sections/specific.txt (hunk 1 above)
Comment (Alex-302) [reply RussianFilter/sections/specific.txt]: Не вижу второго элемента в коде.

**Comment by stanislav-atr** [comment]
I've checked the issue again and only found this element
<details><summary>Screenshot:</summary>

image
</details><br/>
so i swapped the rule to 
`#?#.main__content > section.stories-feed__message:has(> div[data-sid^="partners_widget_horizontal_"])`
to remove the leftover for both extension and desktop, Chrome + Firefox.

**Review by Alex-302 — APPROVED** [review]

File: RussianFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -425,6 +425,7 @@ ideaprog.download#?#.right-content > div.topics:first-child:has(> ul > p.promo)
 dpchas.com.ua#?##header > nav:has(> div.container > section > div.top-reklama)
 comandir.com#?##dle-content > noindex:has(> div.hblock >h4:contains(Новости партнеров))
 5-tv.ru#?#h3:contains(Новости Лентаинформ)
+pikabu.ru#?##?#.main__content > section.stories-feed__message:has(> div[data-sid^="partners_widget_horizontal_"])
```
Comment (Alex-302) [inline RussianFilter/sections/css_extended.txt:428]: invalid rule

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**